pkg/api/getambassador.io/v2/crd_ratelimitservice.go

```diff
@@ -39,6 +39,12 @@ type RateLimitServiceSpec struct {
 
 	// +k8s:conversion-gen:rename=StatsName
 	V3StatsName string `json:"v3StatsName,omitempty"`
+
+	// +k8s:conversion-gen:rename=FailureModeDeny
```

Comment:
this is more of a clarifying question.

I am assuming this conversion is needed so it aligns with what we have defined for v3?

Reply:
Yes, correct. Because we use V2 as our storage version when they are applied it is converted from v3alpha1 to v2. We need a field on the V2 struct to hold this data. We use the convention of adding a V3 at the beginning to indicate it is a v3 only field for storage. The conversion-gen makes sure the generated code will copy the contents of `V3FailureModeDeny` into `FailureModeDeny` and vice-versa.